internal/events/persist_batch.go

```diff
@@ -47,6 +55,29 @@ func (em *eventManager) persistBatchFromBroadcast(ctx context.Context /* db TX c
 	return valid, err
 }
 
+func (em *eventManager) isRootOrgBroadcast(batch *fftypes.Batch) (bool, error) {
```

Comment:
This feels...weird. Should we be tagging these messages in some special way? Unpacking all the data of every message in every batch to look for this special situation just feels very brute-force.